src/src/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -235,7 +235,7 @@ public void removeAll() {
             }
         }
 
-        return tokens.iterator();
+        return tokens;
```

Comment:
I think you got confused by argument to ArrayList constructor. It's not the size of ArrayList, it's capacity (how much memory to allocate in advance).
But if you try to check tokens.size(), you'll get 0. There are no elements in the list (no null elements there)